docs/src/pages/components/icons/icons.md

```diff
@@ -40,28 +40,62 @@ The configuration of a Babel plugin is encouraged.
 
 Each icon also has a "theme": `Filled` (default), `Outlined`, `Rounded`, `Two tone` and `Sharp`. If you want to import the icon component with a "theme" different than default, append the "theme" name to the icon name. For example `@material-ui/icons/Delete` icon with:
 
-- `Filled` "theme" (default) is exported as `@material-ui/icons/Delete`,
-- `Outlined` "theme" is exported as `@material-ui/icons/DeleteOutlined`,
-- `Rounded` "theme" is exported as `@material-ui/icons/DeleteRounded`,
-- `Two tone` "theme" is exported as `@material-ui/icons/DeleteTwoTone`,
-- `Sharp` "theme" is exported as `@material-ui/icons/DeleteSharp`.
+- `Filled` "theme" (default) is exported as *@material-ui/icons/Delete*,
+- `Outlined` "theme" is exported as *@material-ui/icons/DeleteOutlined*,
+- `Rounded` "theme" is exported as *@material-ui/icons/DeleteRounded*,
+- `Two tone` "theme" is exported as *@material-ui/icons/DeleteTwoTone*,
+- `Sharp` "theme" is exported as *@material-ui/icons/DeleteSharp*.
```

Comment:
If anything, we should remove the back-ticks around the theme name, and the quotes from around theme (we've already established in the previous paragraph that "theme" is oddly named). I din't think we should remove the backticks from the export / import.